site2/docs/reference-configuration-bookkeeper.md

```diff
@@ -0,0 +1,585 @@
+# BookKeeper
+
+BookKeeper is a replicated log storage system that Pulsar uses for durable storage of all messages.
+
+## Optional
```

Comment:
@gaoran10 are the configurations of `BookKeeper, client, log4j, log4j shell, standalone, ZooKeeper` all optional?